RomeALincei/ANLcr15.xml

```diff
@@ -0,0 +1,854 @@
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="ANLcr15" xml:lang="en" type="mss">
+    <teiHeader xml:base="https://betamasaheft.eu/">
+        <fileDesc>
+            <titleStmt>
+                <title>Gadla Giyorgis, Taʾammǝra Giyorgis</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="DR"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine
+                    multimediale Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                        licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS0335ANL"/>
+                        <idno>Conti Rossini 15</idno>
+                    </msIdentifier>
+                    
+                    <msContents>
+                        <summary/>
+                        
+                        <msItem xml:id="ms_i1">
+                            <locus from="2ra" to="77va"/>
+                            <title ref="LIT6172DossierGeorge" type="complete"/>
+                        
+                        <msItem xml:id="ms_i1.1">
+                            <locus from="2ra" to="33vb"/>
+                            <title ref="LIT1455GadlaG" type="complete"/>
+                            <note>The beginning of the text seems to be different from other
+                                manuscripts transmitting it.</note>
```

Comment:
I am not sure that the way you formulate it is good. Maybe give references to other manuscripts you mean?